### PR TITLE
added taiko to the Projects working on zkEVMs

### PR DESCRIPTION
**Pull request by seunbayo**
## Description
added taiko to the Projects working on zkEVMs

## Related Issue
#9919

### CODE REVIEW

**Comment by seunbayo** [comment]
@corwintines

**Comment by seunbayo** [comment]
@jmcook1186  can you review it?

**Review by corwintines — CHANGES_REQUESTED** [review]
Thanks @seunbayo!

https://github.com/ethereum/ethereum-org-website/issues/new?assignees=&labels=feature+%3Asparkles%3A%2Ccontent+%3Afountain_pen%3A&template=suggest_layer2.yaml&title=Suggest+a+Layer+2

Need an issue to go with this listing for a layer 2. Can you fill out the above issue template for layer 2 products? After that, just link the issue in this PR. 

I've converted this to draft for now until this is done. After you link the PR, feel free to take this out of draft.

File: src/content/developers/docs/scaling/zk-rollups/index.md [hunk 1]
```diff
@@ -240,6 +240,8 @@ Projects working on zkEVMs include:
 
 - **[Polygon zkEVM](https://polygon.technology/solutions/polygon-zkevm)** - _is a decentralized ZK Rollup on the Ethereum mainnet working on a zero-knowledge Ethereum Virtual Machine (zkEVM) that executes Ethereum transactions in a transparent way, including smart contracts with zero-knowledge-proof validations._
 
+- **[Taiko](https://taiko.xyz)** - _Taiko is building a fully decentralized, Ethereum-equivalent ZK-rollup._
```
Comment (d1onys1us) [inline src/content/developers/docs/scaling/zk-rollups/index.md:243]:
```suggestion
- **[Taiko](https://taiko.xyz)** - _Taiko is a fully decentralized, Ethereum-equivalent ZK Rollup._
```

**Comment by d1onys1us** [comment]
> Thanks @seunbayo!
> 
> https://github.com/ethereum/ethereum-org-website/issues/new?assignees=&labels=feature+%3Asparkles%3A%2Ccontent+%3Afountain_pen%3A&template=suggest_layer2.yaml&title=Suggest+a+Layer+2
> 
> Need an issue to go with this listing for a layer 2. Can you fill out the above issue template for layer 2 products? After that, just link the issue in this PR.
> 
> I've converted this to draft for now until this is done. After you link the PR, feel free to take this out of draft.

hey @corwintines and @seunbayo , I have filled out the form here! https://github.com/ethereum/ethereum-org-website/issues/9919

Thank you and please let me know how I can help or if anything else needs to be done.

**Comment by d1onys1us** [comment]
hey @corwintines are there any changes still requested or is this good to go now? thank you :)

**Review by d1onys1us — APPROVED** [review]

**Comment by d1onys1us** [comment]
hello @corwintines, just a friendly poke. i want to know is there anything else we need to do to merge the PR? we did create the issue, and we are on l2beat. thank you.

**Pull request closed without merging**